rp.js

```diff
@@ -0,0 +1,18 @@
+
+const playgame = function play();
+    const userChoice = prompt(A, B, C)
+    const computerChoice = math.randon(A, B, C)
```

Comment:
the correct way to write this function is `Math.random()` .

you can use `Math.floor()` with  `Math.random()` to get a random integers.
- first use `Math.random()` to get a random number from `0 to 0.99..` 
- if you multiply it with 3 `Math.random()*3` you will get random numbers from `0 to 2.99..` .
- and if you use `Math.floor()` it will give you integer that equal or less than the given number so `Math.floor(2.987)` will return `2`.
- so if you write this `Math.floor( Math.random() * 3)` you will get a random number out of these numbers `0`, `1`, and `2`.